x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -79,8 +82,12 @@ export const movingAverageOperation: OperationDefinition<
       window: [(layer.columns[columnId] as MovingAverageIndexPatternColumn).params.window],
     });
   },
-  buildColumn: ({ referenceIds, previousColumn, layer }) => {
+  buildColumn: (
+    { referenceIds, previousColumn, layer },
+    columnParams = { window: WINDOW_DEFAULT_VALUE }
```

Comment:
We have a default value for window, do we want to keep the named argument in the formula mandatory? I guess it would be OK to omit it and fall back to this default